test/runtests.jl

```diff
@@ -8,6 +12,13 @@ maintainers = ["juliohm"]
 istravis = "TRAVIS" ∈ keys(ENV)
 ismaintainer = "USER" ∈ keys(ENV) && ENV["USER"] ∈ maintainers
 datadir = joinpath(@__DIR__,"data")
+mkpath(datadir)
+
+# download and setup data dependencies
+register(DataDep("juraset", "A geochemical dataset from the Swiss Jura",
+      "https://github.com/cran/compositions/raw/master/data/juraset.rda"))
+dataset = RData.load(joinpath(datadep"juraset", "juraset.rda"))
+CSV.write(joinpath(datadir,"juraset.csv"), dataset["juraset"])
```

Comment:
I think it is possible to specify the directory where the data will be downloaded with this environment variable, but I am not sure: https://www.oxinabox.net/DataDeps.jl/stable/z10-for-end-users/#Moving-Data-1 It would be simpler I guess than downloading and then writing it to the `data` subdirectory.

Could you try to prepend the `test/data` dir to the env variable to see if it works?

Reply:
Hmm.. not sure that I follow you.
The data is downloaded as a .rda file, so it has to be loaded and converted to .csv to be readable by the `readcoda` function.
Are you suggesting to download the .rda file to the test/data directory?

Reply:
I`ll open up another pull request to be sure there is no trace of the dataset.

Reply:
Opened #12.

Reply:
Thank you for being understanding. It is been great to work together. :)